eng/tools/mgmtreport/template.go

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package main
```

Comment:
Could you please refer `generator` tool to use text file as the template, not go file, or you could use some Go template system?